roborazzi/src/main/java/com/github/takahirom/roborazzi/Roborazzi.kt

```diff
@@ -609,10 +609,15 @@ private fun saveOrCompare(
         )
       compareCanvas.release()
 
-      val actualFile = File(
-        goaldenFile.parent,
-        goaldenFile.nameWithoutExtension + "_actual." + goaldenFile.extension
-      )
+      val actualFile = if (roborazziRecordingEnabled()) {
+        // If record option is enabled, we should save the actual file as the golden file.
+        goaldenFile
+      } else {
+        File(
+          goaldenFile.parent,
```

Comment:
What ends up happening if you are using `VerifyAndRecord` without a baseline to feed in?

Reply:
In that scenario, the test fails and generates an output image file and comparison file. I believe this is an inevitable result when we use the verify task, considering its purpose.